src/pages/SudoPrompt.vue

```diff
@@ -9,15 +9,24 @@
     <p>
       Rancher Desktop requires administrative credentials ("sudo access") in
       order to provide a better experience.  We would like to have access for
-      the following:
+      the following reasons:
     </p>
-    <ul>
-      <li v-for="item in explanations" :key="item" class="monospace" v-text="item" />
+    <ul class="reasons">
```

Comment:
Can we reduce the padding to the left of this `ul`? We can provide more contents to the contents inside of this container if we reign this in a bit.

Reply:
Dropped the left padding in 93be988fc29fd4dab6a545b03f86460a75d38993.